sentry-android-distribution/src/main/java/io/sentry/android/distribution/Distribution.kt

```diff
@@ -1,3 +1,70 @@
 package io.sentry.android.distribution
 
-public class Distribution {}
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import io.sentry.android.distribution.internal.DistributionInternal
+import java.util.concurrent.CompletableFuture
+
+/**
+ * The public Android SDK for Sentry Build Distribution.
+ *
+ * Provides functionality to check for app updates and download new versions from Sentry's preprod
+ * artifacts system.
+ */
+public object Distribution {
+  /**
+   * Initialize build distribution with the provided options. This should be called once per
+   * process, typically in Application.onCreate().
+   *
+   * @param context Android context
+   * @param options Configuration options for build distribution
+   */
+  public fun init(context: Context, options: DistributionOptions) {
```

Comment:
I don't know how well the API surface matches the rest of the Sentry SDK but if it looks good to one of the Sentry folks then fine by me.

Reply:
Updated with this